README.md

```diff
@@ -148,8 +126,7 @@ Install this project with:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
- [OPTIONAL] As mentioned by the previous reviewer, I think it could be better if you remove some unnecessary comments in your README file like this one ☝️ you don't need it 🙃 😉 , and the comment at line 145 😊.